.github/workflows/conventional-commits.yaml

```
@@ -0,0 +1,22 @@
+name: Conventional Commits
+
+on:
+  #  https://github.com/marketplace/actions/semantic-pull-request#event-triggers:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

Only as other actions are specifying 22.04.